pkg/server/serverpb/admin.proto

```diff
@@ -548,6 +547,36 @@ message ReplicaMatrixResponse {
   map<int64, ZoneConfig> zone_configs = 2 [(gogoproto.nullable) = false];
 }
 
+message QueriesRequest {
+  enum OrderByCol {
+    SvcLatAvg = 0;
+    Count = 1;
+  }
+
+  enum OrderDir {
+    DESC = 0;
+    ASC = 1;
+  }
+
+  OrderByCol order_by_col = 1;
+  OrderDir order_dir = 2;
+}
+
+message QueriesResponse {
+  message Query {
```

Comment:
Note: should probably use this existing proto https://github.com/cockroachdb/cockroach/blob/master/pkg/roachpb/app_stats.proto#L21

…which I didn't notice when first building this.